deploy/packages/external-libraries.yaml

```
@@ -50,6 +50,7 @@ packages:
       - spark@2.4.0+hadoop
       - sqlite@3.23.1
       - symengine@0.3.0
+      - timemory+mpi~cuda~cupti~caliper~gperftools~python@master
```

```suggestion
      - timemory+mpi+cuda+cupti+caliper~gperftools~python@master
```
cuda/cupti and caliper will be already helpful in coreneuron.